systems/plants/RigidBody.m

```diff
@@ -310,7 +310,11 @@ function dofnum(obj)
       % class. (maybe I should move it up to there?)
       fn = fieldnames(body.param_bindings);
       for i=1:length(fn)
-        body.(fn{i}) = double(subs(body.param_bindings.(fn{i}),poly,pval));
+        if isnumeric(pval)
+          body.(fn{i}) = double(subs(body.param_bindings.(fn{i}),poly,pval));
```

Comment:
the double was only there to make things numeric if they were not.  clearly the original code was not written to support using setParams to pass in a substitution of symbolic variables.  is that what you're trying to accomplish?